en/config_mc/filter_tuning.md

```diff
@@ -35,24 +35,57 @@ Below we look at the impact of the low pass filters.
 
 ## Filters
 
-This is the filtering pipeline for the controllers in PX4:
+The filtering pipeline for the controllers in PX4 is described below.
 
-- On-chip DLPF for the gyro sensor.
-  This is disabled on all chips where it can be disabled (if not, the cutoff frequency is set to the highest level of the chip).
-- A notch filter on the gyro sensor data that is used to filter out narrow band noise, for example harmonics at the rotor blade pass frequency.
-  This filter can be configured using [IMU_GYRO_NF0_BW](../advanced_config/parameter_reference.md#IMU_GYRO_NF0_BW) and [IMU_GYRO_NF0_FRQ](../advanced_config/parameter_reference.md#IMU_GYRO_NF0_FRQ).
-- Low-pass filter on the gyro sensor data.
-  It can be configured with the [IMU_GYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_GYRO_CUTOFF) parameter.
+::: info
+Sampling and filtering is always performed at the full raw sensor rate (commonly 8kHz, depending on the IMU).
+:::
+
+### On-chip DLPF for the Gyro Sensor
+
+This is disabled on all chips where it can be disabled (if not, cutoff frequency is set to the highest level of the chip).
+
+### Notch Filters
+
+Setups that have a significant lower-frequency noise spike (e.g. due to harmonics at the rotor blade pass frequency) can benefit from using the notch filter to clean the signal before it is passed to the low pass filter (these harmonics have a similar detrimental impact on motors as other sources of noise).
+
+Without the notch filter you'd have to set the low pass filter cutoff much lower (increasing the latency) in order to avoid passing this noise to the motors.
```

Comment:
```suggestion
Without the notch filter you'd have to set the low pass filter cutoff much lower (increasing the phase lag) in order to avoid passing this noise to the motors.
```